cmd/typed.go

```diff
@@ -10,12 +10,16 @@ import (
 	"github.com/tendermint/starport/templates/typed"
 )
 
+func init() {
+	typedCmd.Flags().StringVarP(&appPath, "path", "p", "", "path of the app")
```

Comment:
we have the same line here twice for `serve` and `type`. it seems like a duplication for now but will make more sense in future once we complete refactoring the `cmd` pkg.